OpenRA.Mods.Common/Traits/Crates/Crate.cs

```diff
@@ -40,7 +40,7 @@ public class CrateInfo : ITraitInfo, IPositionableInfo, Requires<RenderSpritesIn
 
 		bool IOccupySpaceInfo.SharesCell { get { return false; } }
 
-		public bool CanEnterCell(World world, Actor self, CPos cell, Actor ignoreActor = null, bool checkTransientActors = true)
+		public bool CanEnterCell(World world, Actor self, CPos cell, Actor ignoreActor = null, bool checkTransientActors = true, SubCell subCell = SubCell.FullCell)
```

Comment:
Likewise here - we should be passing this to `GetAvailableSubCell` and from there to `GetActorsAt` instead of ignoring the subcell.